policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/childless/max_age.yaml

```diff
@@ -0,0 +1,9 @@
+description: Indiana has a state EITC that used this maximum age for childless eligiblity.
+metadata:
+  unit: float
+  label: IN EITC maximum age for childless eligibility
```

Comment:
```suggestion
  label: Indiana EITC maximum age for childless eligibility
```

Reply:
Replaced IN with Indiana in all six childless parameter files.